AsyncDisplayKit/ASEditableTextNode.mm

```diff
@@ -143,6 +143,7 @@ - (void)didLoad
   _textKitComponents.textView.accessibilityHint = _placeholderTextKitComponents.textStorage.string;
   configureTextView(_textKitComponents.textView);
   [self.view addSubview:_textKitComponents.textView];
+  [self _updateDisplayingPlaceholder];
```

Comment:
Thanks for the fix!  In general I think it would be better to call this method before addSubview:, as it's possible UITextView does some work upon being added to the hierarchy that is affected by the placeholder stuff (although unlikely, just as a general practice it's good to have stuff set up before adding to hierarchy).